### DISCUSSION TITLE
Will we update to 3.4.0?

### DISCUSSION THREAD

**Discussion by PkllonG** [TrinityCore/TrinityCore#28324] [General]
Better game engine, better game picture! You can also use 64 bit

**Comment by funjoker** [comment]
https://github.com/TrinityCore/TrinityCore/tree/wotlk_classic
feel free to help :)
Currently we are working on a fork
https://github.com/mdX7/TrinityCore/commits/wotlk_classic

**Reply by PkllonG** [reply]
thank you

**Reply by alex397832506** [reply]
Is it working now?

**Reply by funjoker** [reply]
Login works.
Lots of stuff broken tho

**Comment by Aokromes** [comment]
3.4.0 is not real wotlk, is a mix between 3.0 and 2.4.3

**Reply by PkllonG** [reply]
thank you

**Reply by alex397832506** [reply]
> 3.4.0 is not real wotlk, is a mix between 3.0 and 2.4.3

The biggest problem is that there are no clients that can work properly with macOS

**Comment by funjoker** [comment]
https://github.com/TrinityCore/TrinityCore/tree/wotlk_classic supports 3.4.3.52237.
It's very raw and DB is no updated yet (crashes)
